span.go

```diff
@@ -85,9 +85,10 @@ func (e *traceExporter) convertSpan(s *trace.SpanData) *ddSpan {
 }
 
 const (
-	samplingPriorityKey  = "_sampling_priority_v1"
-	statusDescriptionKey = "opencensus.status_description"
-	spanNameKey          = "span.name"
+	keySamplingPriority     = "_sampling_priority_v1"
+	keyStatusDescription    = "opencensus.status_description"
+	keySpanName             = "span.name"
+	keySamplingPriorityRate = "_sampling_priority_rate_v1"
```

Comment:
Not sure I get the impact of renaming `_sampling_priority_v1` to `_sampling_priority_rate_v1`. What's the purpose of adding `_rate` here ?

Reply:
You might have misread it. There is no renaming, they are two separate metrics here:
* `_sampling_priority_v1` (it's the first one in this constant group), defined [here](https://github.com/DataDog/datadog-trace-agent/blob/c973a376fa75663e6f6ac4f7cead0590603616c0/internal/agent/span.go#L21) in the agent.
* `_sampling_priority_rate_v1` defined [here](https://github.com/DataDog/datadog-trace-agent/blob/36d48d1a83cb08101056216f127fcf39bddff714/internal/sampler/prioritysampler.go#L26)

I've only renamed the names of the constants to be prefixed with `key*`

Reply:
`_sampling_priority_v1` specifies whether it was sampled or not and `_sampling_priority_rate_v1` specifies the rate at which it was sampled. Although, frankly, only the rate should be good enough since the formula will result in the same outcome everytime for the same trace ID. So it might as well be done only in the agent without having the need to tell the tracers about it, but that's a different story.

Reply:
Thanks for all these insights.